crates/autopilot/src/domain/settlement/trade/mod.rs

```diff
@@ -154,17 +148,19 @@ pub struct Jit {
 /// Fee per trade in a solution. These fees are taken for the execution of the
 /// trade.
 #[derive(Debug, Clone)]
-pub struct ExecutedFee {
-    /// Gas fee spent to bring the order onchain
-    pub network: eth::SellTokenAmount,
+pub struct FeeBreakdown {
+    /// Total fee spent to bring the order onchain (network fee + protocol fee)
+    /// TODO: express in surplus token
+    pub total: eth::SellTokenAmount,
     /// Breakdown of protocol fees. Executed protocol fees are in the same order
     /// as policies are defined for an order.
-    pub protocol: Vec<(eth::SellTokenAmount, fee::Policy)>,
+    pub protocol: Vec<ExecutedProtocolFee>,
```

Comment:
had to fallback to <total, protocol> since now the values are in different tokens.

willing to revert one the "network" is switched to surplus token as well, since this is a final goal and agreement with solver and frontend team.